scss/_buttons.scss

```diff
@@ -70,12 +72,14 @@
 .btn-primary {
   color: #fff;
   text-shadow: 0 -1px 0 rgba(0,0,0,0.15);
-  @include gradient(#8add6d, #60b044);
+  background-color: #60b044;
+  background-image: linear-gradient(#8add6d, #60b044);
   border-color: darken(#60b044, 2%);
 
   &:hover {
     color: #fff;
-    @include gradient(darken(#8add6d, 5%), darken(#60b044, 5%));
+    background-color: darken(#60b044, 5%);
```

Comment:
Color literals like `#60b044` should only be used in variable declarations; they should be referred to via variable everywhere else.